### ISSUE TITLE
Difference between Payload value in 'W&B' and 'FSUIPC Offsets'

### ISSUE THREAD

**Issue by AivlaSoft** [flybywiresim/a32nx#7082]
### Aircraft Version

Development

### Build info

```json
{
  "built": "2022-04-18T06:59:39+00:00",
  "ref": "master",
  "sha": "0b5b20132e94af5320ee22639a2ce293c23e88c8",
  "actor": "tracernz",
  "event_name": "manual",
  "pretty_release_name": "master:0b5b2013"
}
```

### Describe the bug

There's a difference in the payload weights (read by FSUIPC v7.3.3) and the W&B window when loading a SimBrief flight plan.

### Expected behavior

The values which can be read via FSUIPC should provide the same weight as on the W&B page.

### Steps to reproduce

1) Created a flight plan at SimBrief which provides the following weights:
Payload 17.6t,
ZFW 60.1t
FOB 7.2t
TOW 67.1t 

2) Installed latest updates of the A320neo (see build info above).
3) Loaded the flight plan in 'ATSU - AOC Menu - W&B' via Request SEND, and Boarding START.
It displays a payload weight of 17576 kg and a ZFW of 60076 kg which is in accordance with the flight plan from SimBrief.
4) Requested the flight plan also at 'FMGC - Init page 1', switched to 'INIT page 2' and pressed the LSK 1R to get the ZFW/CG. It properly displays 60.1/24.2 on the scratchpad which was then transferred to the upper right position by pressing LSK 1R again.

5) On the flyPad I also loaded the flight plan (pressed 'From SimBrief') and entered the FOB of 7200 kg. On the upper ECAM it properly indicates 7200 kg. Then entered the Block Fuel 7.2 on the INIT page 2 (LSK 2R).

6) Reading the paylod weights via FSUIPC which provides the following values:

0x13FC Number of payload stations: 10
0x1400 Weight in lbs at station  1: 6105
0x1430 Weight in lbs at station  2: 7770
0x1460 Weight in lbs at station  3: 8695
0x1490 Weight in lbs at station  4: 8695
0x14C0 Weight in lbs at station  5: 0
0x14F0 Weight in lbs at station  6: 0
0x1520 Weight in lbs at station  7: 0
0x1550 Weight in lbs at station  8: 0
0x1580 Weight in lbs at station  9: 1
0x15B0 Weight in lbs at station 10: 1

Payload sum is: 31267 lbs which is 14182 kg which is -3394 kg compared to what is indicated in the 'ATSU - AOC Menu - W&B' window.

**Comment by sidnov** [comment]
Known issue with LBS conversions that will be fixed with #7078

**Comment by aguther** [comment]
Duplicate of #6802
